types/events.go

```diff
@@ -223,6 +223,11 @@ func toBytes(i interface{}) []byte {
 
 // Common event types and attribute keys
 var (
+	EventTypeTx = "tx"
+
+	AttributeKeyAccountSequence = "acc_seq"
+	AttributeKeySignature       = "signature"
```

Comment:
Why not adding it under other `AttributeKey*`?

Reply:
These attributes are for `tx.*`, the attributes below are for `message.*`